### ISSUE TITLE
Feature: implement timeoutlen

### ISSUE THREAD

**Issue by wilmoz** [vimperator/vimperator-labs#339]
This will be very useful for leader key.
Thanks for the plugin is great.

**Comment by gkatsev** [comment]
Yep, that sounds like a good idea. PRs are open. It'll unlikely end up getting implemented otherwise :).
